### PR TITLE
requests are using timeout

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: atomic_reactor/util.py [hunk 1]
```diff
@@ -994,6 +994,26 @@ def get_field(self, field_name, args, kwargs):
...
 
 
+class SessionWithTimeout(requests.Session):
+    """
+    requests Session with added timeout
+    """
+    def __init__(self, *args, **kwargs):
+        super(SessionWithTimeout, self).__init__(*args, **kwargs)
+
+    def get(self, *args, **kwargs):
```
Comment (vrutkovs) [inline atomic_reactor/util.py]: Wouldn't it be easier to override just `def request` there?

Comment (rcerven) [reply atomic_reactor/util.py]: not sure

Comment (lcarva) [reply atomic_reactor/util.py]: That's worth investigating.

Comment (rcerven) [reply atomic_reactor/util.py]: changed to override just request

**Comment by vrutkovs** [comment]
Release notes updated

**Pull request merged**